### PR TITLE
Add model management related configuration

### PR DESCRIPTION
**Pull request by aar0nTw**
image-tag: ~`ch16758-7ccf11c6`~ ~`ch16758-639699db`~ ~`ch16758-9acac206`~ ~`ch16758-9d830cb2`~ ~`ch16758-4fb1d40a`~ `ch16758-4a268843`

### CODE REVIEW

File: packages/graphql-server/src/resolvers/attr.ts [hunk 1]
```diff
@@ -108,8 +109,8 @@ export class Attributes {
       const customTransform = value.deserialize;
       const transform = customTransform || typeTransform || noop;
       // admins transform
```
Comment (ctiml) [inline packages/graphql-server/src/resolvers/attr.ts:111]: Could we change this comment? or remove it?

**Comment by popcornylu** [comment]
The group settings should be only visible by group admin. And when i click the job and model tab, it would lead me to home page.
image

**Comment by popcornylu** [comment]
Could we change the `learn more` color to white.

image

**Comment by popcornylu** [comment]
A group admin without system admin permission, he cannot configure the MLFlow settings.

**Comment by popcornylu** [comment]
In `ch16758-4fb1d40a`, the non-system admin users can query all group information by this query
```
# Write your query or mutation here
{
  me {
    isAdmin
    groups {
      name
      mlflow {
        trackingUri
        uiUrl
      }
    }
  }
  groups {
    id
    name
    admins
    mlflow {
      trackingUri
      uiUrl
    }
  }
}	
```

**Comment by popcornylu** [comment]
LGTM

**Pull request merged**